tests/pipeline_backend_test.py

```diff
@@ -24,15 +25,16 @@
 import pipeline_dp
 import pipeline_dp.combiners as dp_combiners
 from pipeline_dp import DataExtractors
-from pipeline_dp.pipeline_backend import BeamBackend
+from pipeline_dp.beam_backend import BeamBackend
```

Comment:
Could you please move BeamBackendTest into corresponding file? (it's common to use 1:1 correspondence between py file and its tests)